routes/celebrities.js

```diff
@@ -0,0 +1,64 @@
+const router = require('express').Router();
+const Celebrity = require('../models/celebrity');
+
+router.get('/celebrities', (req, res, next) => {
+  Celebrity.find()
+    .then(celebrities => {
+      res.render('celebrities/index', { celebrityList: celebrities });
+    }).catch(err => {
+      next(err);
+    });
+});
+
+router.post('/celebrities', (req, res) => {
```

Comment:
If the base path in "app.use" is "/celebrities", then the path defined here would be "/".